tests/integration/targets/homebrew/tasks/formulae.yml

```diff
@@ -321,3 +321,45 @@
         - "package_result.msg == 'Changed: 0, Unchanged: 2'"
         - "package_result.changed_pkgs == []"
         - "package_result.unchanged_pkgs | sort == ['gnu-tar', 'gnu-time']"
+
+# Test alias handling with sqlite (that is aliased to sqlite3)
+- block:
+  - name: Make sure sqlite package is not installed
+    homebrew:
+      name: "sqlite"
+      state: absent
+      update_homebrew: false
+    become: true
+    become_user: "{{ brew_stat.stat.pw_name }}"
+
+  - name: Install sqlite package using alias (sqlite3)
+    homebrew:
+      name: "sqlite3"
+      state: present
+      update_homebrew: false
+    become: true
+    become_user: "{{ brew_stat.stat.pw_name }}"
+    register: package_result
+
+  - assert:
+      that:
+        - package_result.changed
+        - "package_result.msg == 'Package installed: sqlite3'"
+        - "package_result.changed_pkgs == ['sqlite3']"
+        - "package_result.unchanged_pkgs == []"
+
+  - name: Again install sqlite package using alias (sqlite3)
+    homebrew:
+      name: "sqlite3"
+      state: present
+      update_homebrew: false
+    become: true
+    become_user: "{{ brew_stat.stat.pw_name }}"
+    register: package_result
```

Comment:
I would recommend using a variable name different from the one used in the previous task (line 342), so it becomes clearer to whoever is reading what exactly is being referred to here.